### ISSUE TITLE
More dead links

### ISSUE THREAD

**Issue by robmarkcole** [robmarkcole/satellite-image-deep-learning#16]
[✖] https://github.com/aws-samples/amazon-rekognition-video-analyzer → Status: 429
415
[✖] https://github.com/RemotePixel/remotepixel-api → Status: 429
416
[✖] https://github.com/zarr-developers/zarr → Status: 429
417
[✖] https://github.com/PacktPublishing/-Practical-Deep-Learning-on-the-Cloud → Status: 429
418
[✖] https://github.com/jrosebr1/simple-keras-rest-api → Status: 429
419
[✖] https://github.com/pytorch/serve → Status: 429
420
[✖] https://github.com/ryfeus/amazon-rekognition-custom-labels-satellite-imagery → Status: 429
421
[✖] https://github.com/developmentseed/chip-n-scale-queue-arranger → Status: 429
422
[✖] https://github.com/developmentseed/fastai-serving → Status: 429
423
[✖] https://github.com/Esri/raster-deep-learning → Status: 429

**Comment by robmarkcole** [comment]
Error 429 appears to be a red herring, raised when there are too many requests by the API

**Comment by robmarkcole** [comment]
Simply rerunning the jobs was enough to clear the 429 error
